api/tests/unit_tests/core/app/segments/test_factory.py

```diff
@@ -29,7 +30,7 @@ def test_integer_variable():
 
 
 def test_float_variable():
-    test_data = {"value_type": "number", "name": "test_float", "value": 3.14}
+    test_data = {"value_type": "number", "name": "test_float", "value": math.pi}
```

Comment:
I don’t think it makes sense xD

Reply:
Fixed, by ingoring the FURB152  math-constant rule.